core/src/test/java/test/org/apache/spark/JavaAPISuite.java

```diff
@@ -280,15 +280,15 @@ public void sortBy() {
 
   @Test
   public void foreach() {
-    LongAccumulator accum = sc.sc().longAccumulator();
```

Comment:
is it recommended to get `SparkContext` from `JavaSparkContext` and call APIs? cc @zsxwing @srowen @JoshRosen